buffalo/cmd/new.go

```diff
@@ -201,6 +201,12 @@ var newCmd = &cobra.Command{
 		}
 		run.WithGroup(gg)
 
+		g := genny.New()
+		g.Command(exec.Command("go", "mod", "tidy"))
```

Comment:
I'm not sure if what I'm about to say goes here but:
After testing this PR on Codespaces (and locally), `go.mod` still contains old versions (Buffalo 0.15.5 instead of 0.16.27, etc.).
I mean, it solves the things one had to run manually after a `buffalo new`, but maybe after a new release it would be helpful to check out these versioning things.

PS: I'm guessing it's because of the `go get github.com/gobuffalo/buffalo-pop/v2` (which adds an indirect dependency, but the `2.3.0` version depends on an old Buffalo, or idk, don't know a lot about how Go handles dependencies).

Only thing that worked for me in order to use latest is:

```bash
rm go.*
go mod init [project]
go mod tidy
```

Reply:
Good point @bryanjhv. I think we need to make sure that works before merging this PR. Thanks for the heads up.

Reply:
While running `buffalo new app` I did find the following error:

# github.com/gobuffalo/buffalo-pop/v2
/usr/local/go/pkg/tool/darwin_arm64/link: /usr/local/go/pkg/tool/darwin_arm64/link: combining dwarf failed: unknown load command 0x80000034 (2147483700)

Reply:
same problem!

Reply:
Which Go version are you using?
I tested it on Linux and everything went smooth.
Maybe it's specific to Mac OS? Maybe another regression?
https://github.com/golang/go/issues/25908

Reply:
Yes. Agree, this seems to be specific to apple M1 chips.

Reply:
Concerning the topic of updating direct dependencies after creating a new application with `buffalo new`: I added this functionality in the following PR: https://github.com/gobuffalo/buffalo/pull/2128

Reply:
@fasmat Just merged it. I think this is related with #2130 as we will move the CLI out of the buffalo library. I think that would be best.